### PR TITLE
fix: Remove overflow string in label delete confirm button

### PR DESCRIPTION
**Pull request by memorycancel**
Fix [!7197](https://github.com/chatwoot/chatwoot/issues/7197) [CW-1900]

From
image
To
image

### CODE REVIEW

**Review by vishnu-narayanan — APPROVED** [review]

File: app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue [hunk 1]
```diff
@@ -127,14 +127,10 @@ export default {
     }),
     // Delete Modal
     deleteConfirmText() {
-      return `${this.$t('LABEL_MGMT.DELETE.CONFIRM.YES')} ${
-        this.selectedResponse.title
-      }`;
+      return `${this.$t('LABEL_MGMT.DELETE.CONFIRM.YES')}`;
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue:130]:
```suggestion
      return this.$t('LABEL_MGMT.DELETE.CONFIRM.YES');
```

File: app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue [hunk 2]
```diff
@@ -127,14 +127,10 @@ export default {
...
     },
     deleteRejectText() {
-      return `${this.$t('LABEL_MGMT.DELETE.CONFIRM.NO')} ${
-        this.selectedResponse.title
-      }`;
+      return `${this.$t('LABEL_MGMT.DELETE.CONFIRM.NO')}`;
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/labels/Index.vue:133]:
```suggestion
      return this.$t('LABEL_MGMT.DELETE.CONFIRM.NO');
```

**Comment by iamsivin** [comment]
@memorycancel Kindly take a look at the minor comments that I have included.

**Comment by memorycancel** [comment]
> @memorycancel Kindly take a look at the minor comments that I have included.

@iamsivin  Thanks! Suggestions applied, Please take a look again.

**Review by iamsivin — APPROVED** [review]

**Comment by memorycancel** [comment]
@vishnu-narayanan Can you take a look at this again, thanks!

**Review by vishnu-narayanan — APPROVED** [review]
@memorycancel Thank you for the PR.

**Pull request merged**